### ISSUE TITLE
Incompatable with Bootstrap

### ISSUE THREAD

**Issue by James2k2** [kylefox/jquery-modal#284]
The plugin has an issue whereby the 'modal' class is incompatible with Bootstrap (bootstrap already has a class called modal). a simple rename to something like jq-modal fixes the issue for me. Also, the z-index of the overlay is too low. This should either be customisable as an option, or generally increased. I have fixed these through mods of my own, but for those wanting to just use the basic clean design will run into problems if they use bootstrap or any kind of multi-layered page design.

For those wanting a proper fix whilst maintaining the default style, I guess you could duplicate the default modal CSS and give it a different name, then use modalClass to specify the new name. The only reason I mention this is because Bootstrap is quite popular as a done deal type integration.

**Comment by mesuttascic** [comment]
Yes, it is not compatible with bootstrap. I tried with bootstrap@5.0.0-beta2.
I lost 5-6 hours to find that problem. I don't know how to achieve it.
@James2k2 can you share your fixed version?
Maybe you can pull request too.

**Comment by ChadHorn** [comment]
In the README, the author states "Naming conflict with Bootstrap: Bootstrap's modal uses the same $.modal namespace. If you want to use jquery-modal with Bootstrap, the simplest solution is to manually modify the name of this plugin."

Hope this guidance helps.

**Comment by sumandahal** [comment]
Hey guys, thanks for all the tips. I am not a js expert so unsure where to make the change. Can I do a search for $.modal on the js file and replace that with $.jqmodal, will that fix this? 

Thanks

**Comment by Friends4U** [comment]
I would like to know 2 :)

**Comment by Friends4U** [comment]
> In the README, the author states "Naming conflict with Bootstrap: Bootstrap's modal uses the same $.modal namespace. If you want to use jquery-modal with Bootstrap, the simplest solution is to manually modify the name of this plugin."
> 
> Hope this guidance helps.

How?
